format.c

```diff
@@ -146,6 +146,13 @@ int cmp_rusage_stime(const void *x, const void *y)
 
 
 
+int cmp_double(const void *x, const void *y)
```

Comment:
I suspect we should call this cmp_cputime or similar, for consistency with the other cmp_* functions?